airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -684,7 +693,7 @@ void testCustomDbtTransformations() throws Exception {
     final OperatorDbt dbtConfig = new OperatorDbt()
         .withGitRepoUrl("https://github.com/fishtown-analytics/jaffle_shop.git")
         .withGitRepoBranch("main")
-        .withDockerImage("fishtownanalytics/dbt:0.19.1");
+        .withDockerImage("airbyte/normalization:dev");
```

Comment:
just making sure i follow here...so we are using airbyte normalization because it has the right dbt deps. But when we hit `dbt run` below what are we actually running then? Airbyte normalization?

Reply:
No, it's just using the dbt binary and packages from the docker image, the models are coming from:
`https://github.com/fishtown-analytics/jaffle_shop.git`